pym/bob/input.py

```diff
@@ -2224,6 +2225,23 @@ def validate(self, data):
 
 class RecipeSet:
 
+    BUILD_DEV_SCHEMA = schema.Schema(
+        {
+            schema.Optional('destination') : str,
+            schema.Optional('force') : bool,
+            schema.Optional('no-deps') : bool,
+            schema.Optional('build-only') : bool,
+            schema.Optional('checkout-only') : bool,
+            schema.Optional('clean') : bool,
+            schema.Optional('quiet') : int,
+            schema.Optional('verbose') : int,
```

Comment:
`quiet` and `verbose` could be collapsed into `verbosity`. This is then the default that is incremented by `-v` and decremented by `-q`.